packages/appstore/src/components/fiat-transaction-list/fiat-transaction-list.tsx

```diff
@@ -0,0 +1,147 @@
+
+    const accountName = (is_virtual: boolean, currency: string, is_wallet: boolean) =>
+        `${is_virtual ? localize('Demo') : ''} ${getCurrencyDisplayCode(currency)} ${localize(
+            is_wallet ? 'Wallet' : 'account'
+        )}`;
+
+    const TransactionsForADay = ({
+        day,
+        transactions,
+    }: {
+        day: string;
+        transactions: Required<Statement>['transactions'];
+    }) => {
+        return (
+            <div className='fiat-transaction-list__day'>
+                <Text
+                    size={isMobile() ? 'xxxxs' : 'xxxs'}
+                    line_height={isMobile() ? 'm' : 's'}
+                    color='less-prominent'
+                    weight='lighter'
+                >
+                    {day}
+                </Text>
+                {transactions
+                    .map(transaction => {
+                        if (
+                            transaction.amount === undefined ||
+                            transaction.balance_after === undefined ||
+                            transaction.action_type === undefined
+                        )
+                            return null;
+                        let account_title = wallet_title;
+                        let account_currency = wallet_currency;
+                        let icon = wallet_icon;
+                        let icon_type = 'fiat';
+                        let platform = null;
+                        if (transaction.action_type === 'transfer') {
+                            const other_loginid =
+                                transaction.to?.loginid === loginid
+                                    ? transaction.from?.loginid
+                                    : transaction.to?.loginid;
+                            if (!other_loginid) return null;
+                            const other_account = accounts[other_loginid];
+                            if (other_account) {
+                                if (!other_account.currency) return null;
+                                account_currency = other_account.currency;
+                                account_title = accountName(
+                                    !!other_account.is_virtual,
+                                    other_account.currency,
+                                    other_account.account_category === 'wallet'
+                                );
+                                icon = getWalletCurrencyIcon(
+                                    other_account.is_virtual ? 'demo' : other_account.currency || '',
+                                    is_dark_mode_on,
+                                    false
+                                );
+                                icon_type = isCryptocurrency(account_currency) ? 'crypto' : 'fiat';
+                            } else {
+                                const app_account = linked_accounts.find(account => account?.loginid === other_loginid);
```

Comment:
If the account that is being transferred to/from is not in `client.accounts`, look for it by `loginid` in linked accounts to determine its platform